### PR TITLE
regular-pop-up-menu-cy-229

### PR DESCRIPTION
**Pull request by cris22380**

### CODE REVIEW

**Comment by cris22380** [comment]
- From storybook
Screen Shot 2022-01-03 at 13 44 59

- From Figma
pop up scrollfigma

- From storybook
pop up storybook

- From Figma
pop up figma

File: src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const [value, setValue] = useState("");
+  const [toogleModal, setToogleModal] = useState(false);
+  const onChange = (e) => {
+    setValue(e.target.value);
+  };
+  return (
+    <>
+    {toogleModal ? (
+        <RegularPopUpMenu>
+          <SPopUpHeader>
```
Comment (diegomart2000) [inline src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx]: @cris22380 Styled components should not be exposed.

File: src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            <ModalHeader onClose={() => setToogleModal(false)}>
```
Comment (diegomart2000) [inline src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx]: This is internal implementation. Should not be exposed. These 2 values should be passed as props.

File: src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              <ModalHeaderTitle>Label</ModalHeaderTitle>
+            </ModalHeader>
+          </SPopUpHeader>
+
+          <SPopUpContent>
```
Comment (diegomart2000) [inline src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx]: This is internal implmentation, should not be exposed. Main component should receive child items only.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Comment by diegomart2000** [comment]
@cris22380 follow stories in here https://league-qodxxzzkp-caisy.vercel.app/?path=/story/components-menus-popovers-modal--default also move the stories under this category.

File: src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        </RegularPopUpMenu>
+      ) : (
+        <STooglePopUp onClick={() => setToogleModal(true)}>Click me!!</STooglePopUp>
+    )}
+    </>
+  );
+};
+
+export default {
+  title: "Components/RegularPopUpmenu/PopUp",
```
Comment (diegomart2000) [inline src/components/regular-pop-up-menu/RegularPopUpMenu.stories.tsx]: Move to proper category already in place and make sure name is properly spelled.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

File: src/components/regular-pop-up-menu/RegularPopUpMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { ModalHeader } from "..";
```
Comment (diegomart2000) [inline src/components/regular-pop-up-menu/RegularPopUpMenu.tsx]: @cris22380 avoid these kind of imports. This causes circular references, use named imports

File: src/components/regular-pop-up-menu/styles/SPopUpHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+interface ISPopUpHeader {
+  color?: string;
+}
+
+export const SPopUpHeader = styled.div<ISPopUpHeader>`
+width: 100%;
```
Comment (diegomart2000) [inline src/components/regular-pop-up-menu/styles/SPopUpHeader.tsx]: @cris22380 fix formatting.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**